src/test/java/teammates/test/pageobjects/InstructorCourseStudentDetailsEditPage.java

```diff
@@ -73,10 +70,14 @@ public InstructorCourseStudentDetailsEditPage submitUnsuccessfully(String studen
         if (comments != null) {
             fillTextBox(commentsTextbox, comments);
         }
-        submitButton.click();
-        return this;
+        //If teamName is not changed, there is no need to confirm
```

Comment:
This comment seems a bit redundant... Can you explain the 'why'? e.g. explain why this behaviour is required